apps/docs/src/guide/contracts/minted-token-asset-id.md

```diff
@@ -18,3 +18,11 @@ Imagine that this contract is already deployed and we are about to mint some coi
 <<< @/../../docs-snippets/src/guide/contracts/minted-token-asset-id.test.ts#minted-token-asset-id-2{ts:line-numbers}
 
 Since the asset ID depends on the contract ID, which is always dynamic (unlike the sub ID, which can be set to a fixed value), the helper `getMintedAssetId` can be used to easily obtain the asset ID for a given contract ID and sub ID.
```

Comment:
```suggestion
## Obtaining the Asset ID

Since the asset ID depends on the contract ID, which is always dynamic (unlike the sub ID, which can be set to a fixed value), the helper `getMintedAssetId` can be used to easily obtain the asset ID for a given contract ID and sub ID.
```

Reply:
Thank you for reviewing and suggestion on the changes